aspnetcore/blazor/hosting-model-configuration.md

```diff
@@ -17,11 +17,26 @@ By [Daniel Roth](https://github.com/danroth27)
 
 This article covers hosting model configuration.
 
-<!-- For future use:
-
 ## Blazor WebAssembly
 
--->
+As of the ASP.NET Core 3.2 Preview 3 release, Blazor WebAssembly supports configuration from:
+
+* *wwwroot/appsettings.json*
+* *wwwroot/appsettings.{ENVIRONMENT}.json*
```

Comment:
Should this say `HostingEnvironment` instead of `Environment`? It's referred to as `HostingEnvironment` every where else that follows.

Reply:
Yes ... I was making this match the way that we normally use the placeholder for this file.

I still need to hear back on the question that I ask in the opening comment and then I can fix it all up.